engine/src/main/java/org/hibernate/search/engine/search/predicate/spi/FilterPredicateBuilder.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.engine.search.predicate.spi;
+
+public interface FilterPredicateBuilder extends SearchPredicateBuilder {
```

Comment:
I would rename this class to `NamedPredicateBuilder` (see my main comment).

Reply:
I'll change it.